src/celeritas/ext/GeantImporter.cc

```diff
@@ -564,6 +564,7 @@ import_optical(detail::GeoOpticalIdMap const& geo_to_opt)
         get_property(&optical.rayleigh.compressibility,
                      "ISOTHERMAL_COMPRESSIBILITY",
                      ImportUnits::len_time_sq_per_mass);
+        optical.rayleigh.temperature = material->GetTemperature();
```

Comment:
Instead of this, we can pull it from the MaterialView when we construct our optical materials.